libsql.go

```diff
@@ -223,9 +224,6 @@ func openEmbeddedReplicaConnector(dbPath, primaryUrl, authToken string, readYour
 			}
 		}()
 	}
-	if err != nil {
```

Comment:
Why is this necessary?

Reply:
It isn't *necessary* but I took it out because it's redundant and the compiler will elide it anyway - as shown by the go linter. The error in that scope (at line 201 is already checked for nil (line 203) and the only other subsequent error is in the go routine (220) and never escapes the `if` or, indeed, the go routine.

I can add it back in and re-push if you like?